src/server/views/dynamic/liveblog/highscore.handlebars

```diff
@@ -77,7 +77,7 @@
                 <tr>
                     <th>{{__ 'PLACE'}}</th>
                     <th>{{__ 'TEAM'}}</th>
-                    <th>{{__ 'DISTANCE'}}</th>
+                    <th>{{__ 'DONATION'}}</th>
```

Comment:
Why is this in this PR?

Reply:
Just a detail I noticed. And decided I didn't want to open a PR for a three liner